README.md

```diff
@@ -96,6 +96,12 @@ Add LiveView NPM dependencies in your package.json.
 }
 ```
 
+Make sure you install the new dependency.
+
+```bash
+(cd assets && npm install)
```

Comment:
```suggestion
npm install --prefix assets
```

Reply:
Why is this --prefix better - I struggle to see how ```cd``` can be misunderstood? I woul dhave to look up what --prefix does before running the command...

Reply:
I am familiar with the *expected* behaviour of `--prefix` from an ops perspective to specify where the start.  I am fine with either, @chrismccord any thoughts?

That said, the existing documentation does it in two lines

https://phoenixframework.org/blog/static-assets

```
cd assets
npm install
```

So I am good with  either / any / both